packages/@react-spectrum/icon/docs/Icon.mdx

```diff
@@ -25,12 +25,26 @@ category: Misc
 ---
 
 # Icon
-<p>Docs coming soon</p>
+
+<p>{docs.exports.Icon.description}</p>
 
 ## Available Icons
 
 ### Workflow
-<HeaderTable name={workflowIconPackageData.name} version={workflowIconPackageData.version} />
+
+<HeaderInfo
+  packageData={workflowIconPackageData}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/iconography/'}
+  ]} />
+
+<br />
```

Comment:
Opted for this instead of `####Description` cuz it looks better to me.
This is what it looks like with the description header:
<img width="1087" alt="Screen Shot 2020-06-17 at 11 37 59 AM" src="https://user-images.githubusercontent.com/9661127/84937047-07270b80-b090-11ea-8f51-f396315b638c.png">